test/minikube/minikube_tls_admin_test.go

```diff
@@ -34,15 +34,16 @@ func verifyKeystore(t *testing.T, namespace string, podName string, keystore str
 }
 
 func TestKubernetesTLS(t *testing.T) {
+	if testlib.IsOpenShiftEnvironment(t) {
```

Comment:
Do we still have these problems with the new SCC?

Reply:
Wasn't this a 3.11 specific issue?

Reply:
it might be specific to 3.11. I am still investigating. It has nothing to do with the SCC. It is it a bug in 3.11. I have not retested in 4.3 yet.

Reply:
Maybe we can add a comment in the test why we are skipping them for OCP.